### PR TITLE
Enable direct export of ViTs

### PR DESCRIPTION
**Pull request by sovrasov**
### Summary

Redundant ONNX conversion step is eliminated. This also avoids extra FS usage

### CODE REVIEW

**Comment by sovrasov** [comment]
Need to merge https://github.com/openvinotoolkit/training_extensions/pull/3646 first [done]

**Comment by sovrasov** [comment]
@harimkang please have a look at 5a50259. We have a couple of unexpected outputs on export introduced in https://github.com/openvinotoolkit/training_extensions/commit/e402a84f99187302ce8b72576c3fd668cff3dd4e

File: src/otx/algo/classification/vit.py [hunk 1]
```diff
@@ -140,14 +139,18 @@ def _forward_explain_image_classifier(
...
-        return {
+        outputs = {
             "logits": logits,
             "feature_vector": feature_vector,
             "saliency_map": saliency_map,
-            "scores": scores,
-            "labels": labels,
         }
 
+        if not torch._C._is_tracing():  # noqa: SLF001
```
Comment (eunwoosh) [inline src/otx/algo/classification/vit.py]:
```suggestion
        if not torch.jit.is_tracing():  # noqa: SLF001
```
Is it ok to use as above? If it's, how about avoiding to use internal funciton?

Comment (sovrasov) [reply src/otx/algo/classification/vit.py]: Updated

**Review by eunwoosh — APPROVED** [review]

**Review by kprokofi — APPROVED** [review]

**Pull request merged**